isisd/isis_spf.c

```diff
@@ -1190,14 +1236,46 @@ static void add_to_paths(struct isis_spftree *spftree,
 		   vertex->d_N);
 #endif /* EXTREME_DEBUG */
 
+	if (VTYPE_IS(vertex->type)
+	    && !CHECK_FLAG(spftree->flags, F_SPFTREE_NO_ADJACENCIES)) {
+		if (listcount(vertex->Adj_N) > 0) {
+			if (spftree->type == SPF_TYPE_TI_LFA) {
+				struct isis_adjacency *adj;
+
+				if (isis_lfa_check(spftree, vertex) != 0)
+					return;
+
+				adj = isis_adj_find(area, spftree->level,
+						    vertex->N.id);
+				if (adj)
+					sr_adj_sid_add_single(
+						adj, spftree->family, true,
+						vertex->Adj_N);
+			}
+		} else if (IS_DEBUG_SPF_EVENTS)
+			zlog_debug(
+				"ISIS-Spf: no adjacencies do not install backup Adj-SID for %s depth %d dist %d",
```

Comment:
nit -- Spf/SPF ... and maybe add a comma after "adjacencies"

Reply:
Done. I used a separate commit since "ISIS-Spf" was being used in the whole file.